src/CrossChainReceiverFactory.sol

```diff
@@ -17,7 +17,7 @@ contract CrossChainReceiverFactory is IERC1271, IERC5267, MultiCallContext, TwoS
     CrossChainReceiverFactory private immutable _cachedThis;
     bytes32 private immutable _proxyInitHash;
     string public constant name = "ZeroExCrossChainReceiver";
-    bytes32 private constant _NAMEHASH = 0x819c7f86c24229cd5fed5a41696eb0cd8b3f84cc632df73cfd985e8b100980e8;
+    uint256 private constant _NAMEHASH = 0x819c7f86c24229cd5fed5a41696eb0cd8b3f84cc632df73cfd985e8b100980e8;
```

Comment:
I think this works both ways? It shouldn't matter whether it's declared as `uint256` or `bytes32` if all you're going to do is reference it in Yul. Yul doesn't have types